src/App.js

```diff
@@ -0,0 +1,76 @@
+import LottoGame from "./domain/LottoGame";
+import InputView from "./view/InputView";
+import OutputView from "./view/OutputView";
+import { ERROR_MESSAGE } from "./constants/message";
+import splitByDelimiter from "./utils/splitByDelimiter";
+import { SYMBOL } from "./constants/condition";
+
+class App {
+  #lottoGame;
+
+  constructor() {
+    this.#lottoGame = new LottoGame();
+  }
+
+  async play() {
+    const lottos = await this.#purchaseLottos();
+    const winningLotto = await this.#generateWinningLotto();
+
+    this.#drawLotto(lottos, winningLotto);
+
+    this.#replay();
+  }
+
+  async #purchaseLottos() {
+    try {
+      const purchaseAmount = Number(await InputView.readPurchaseAmount());
+      const lottos = this.#lottoGame.issueLottos(purchaseAmount);
```

Comment:
개인적인 선호인데 조금 더 읽기 쉬운건 이런 방향이라고 생각합니다!

```suggestion
      const purchaseAmount = await InputView.readPurchaseAmount();
      const lottos = this.#lottoGame.issueLottos(Number(purchaseAmount));
```